gr-filter/grc/filter_freq_xlating_fir_filter_xxx.block.yml

```diff
@@ -53,4 +53,13 @@ templates:
     - set_taps(${taps})
     - set_center_freq(${center_freq})
 
+cpp_templates:
+    includes: [ '#include <gnuradio/filter/freq_xlating_fir_filter.h>' ]
+    declarations: 'filter::freq_xlating_fir_filter_${type}::sptr ${id};'
+    make: 'this->${id} = filter::freq_xlating_fir_filter_${type}(${decim}, ${taps}, ${center_freq}, ${samp_rate});'
```

Comment:
Can't instantiate classes directly ... you have to use the `make()` function.
`this->${id} = filter::freq_xlating_fir_filter_${type}::make(...)`
Are you sure you ran this code? It didn't compile until I made this change.

Reply:
I was sure, that I've compiled that code. Maybe I missed something.

What I did was this:
- clone repo,
- new branch with changes
- go into build-dir (was created in another run before already)
- run make

Did I maybe miss something with that? I have set up ccache at my system doe you think it could interfere?

Reply:
What @willcode  is asking for is that you build a flow graph in GRC with this block, (for example, Null Source -> freq xlating fir -> Null Sink), let GRC generate C++ and see whether *that* C++ compiles!

Reply:
I missed that direct instantiation in #6674, as well; my apologies. Could you, once you fixed this, also add a commit to fix #6674 lack of `::make(…)`, please?

Reply:
I missed that one too.

Reply:
Okay, I will add this.

Reply:
@Akira25 Still need to add `make()` to this PR and test when you get a chance.

Reply:
@willcode Apparently there is still something more to do on this class. I will come back with some questions in the next days, hopefully.

Reply:
@Akira25 Yes, you made the change in the other @marcusmueller mentioned but not this one.